explainer.md

```diff
@@ -40,39 +40,24 @@ We propose a new concept, **display locking**, to assist developers with **allev
 jank caused by DOM updates** and having some control on **when to pay rendering costs**,
 while also **allowing things that depend on the updates to get the up-to-date values** when urgent.
 
-Using display locking, the developer will be able to lock an element and its subtree,
-preventing visual updates.
-Then, the developer will be able construct the locked subtree’s DOM however they desire,
-and insert it into the DOM without any rendering cost or jank.
-
-After insertion, we can request the rendering values (style, layout, etc) to be updated.
-These updates can be *co-operative* -- **interleaved with other
-work such as running script or DOM updates outside of the locked subtree** if they are going to take a long time,
-or calculated synchronusly if we need it.
-The developer is also able to commit the element's lock,
-which will calculate the rendering values if it's not up-to-date,
-and cause the visual updates of the modified subtree to appear.
-
-In essence, display locking will
-make it possible to **perform complicated DOM updates without causing the rest of
-the page to jank** and **only pay rendering costs on things that really need it**.
-
 In the rest of the document, we describe the display locking concept in detail.
 We will first examine some motivating examples which we commonly observe in rich
 web applications. We will discuss the intent of each of the examples, as well as
-the areas that can potentially cause jank. In order to understand the display
-locking proposal details, we will then describe common phases in user-agent
-lifecycles. Specifically, we will talk about script, layout, paint, compositing,
-and presentation to the screen as well as areas that display locking is geared
-to improve. We will then move on to describe the API changes required to
+the areas that can potentially cause jank.
+We will then move on to describe the API changes required to
 implement display locking, with a description of each new functionality. With
 that in place, we will revisit the motivating examples to see how display
 locking can be applied in order to improve their performance. Finally, we will
-discuss possible display locking implementations, along with possible corner
-cases and limitations.
+discuss possible corner cases and limitations.
+
+In order to understand the display locking proposal details, a brief explanation
+of common phases in user-agent lifecycles is available [here](./lifecycle.md).
```

Comment:
Will "./lifecycle.md" pick up the correct thing when this is merged?

Reply:
Yep! Tested on the add-size branch.